src/map/skill.cpp

```diff
@@ -14031,7 +14028,7 @@ TIMER_FUNC(skill_castend_id){
 					}
 				}
 				break;
-			case WE_CALLBABY:
+		
```

Comment:
Revert this as it doesnt belong to any of the methods we're migrating (see skill_impl.hpp)